www/docs/content/development/events/events-list.md

```diff
@@ -705,6 +705,53 @@ Object of the following format:
 
 ---
 
+## Discount Events
+
+This section holds all events related to discounts.
+
+<table class="reference-table">
+<thead>
+<tr>
+<th>
+Event Name
+</th>
+<th>
+Description
+</th>
+<th>
+Event Data Payload
+</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td>
+
+`discount.created`
+
+</td>
+<td>
+
+Triggered when a discount is created.
+
+</td>
+<td>
+
+Object of the following format:
+
+```js noReport noCopy
+{
+  id // string ID of discount
+}
+```
+
+</td>
+</tr>
+</tbody>
+</table>
+
```

Comment:
@shahednasser could you look at this please, if it should be in that pr or if you want to handle it

Reply:
@Arsenalist let's remove it for now, as it will appear in the docs once the PR is merged, even if it's not released yet. I'll add it in after the release